build.sbt

```diff
@@ -7,6 +7,18 @@ lazy val backend = (project in file("backend"))
   .dependsOn(core % "test->test;compile->compile")
   .settings(backendSettings:_*)
 
+lazy val localBackend = (project in file("default-backends/local"))
```

Comment:
I think this is fine for now but at some point we might want to think about also breaking this stuff out a bit